src/components/Common/SearchInputBox/SearchInputBox.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useRef } from "react";
+import SearchInputBox from "./SearchInputBox";
+
+export default {
+    title: "Components/Search/SearchInputBox",
+    component: SearchInputBox,
+};
+
+const Template = (args) => {
+    const [searchValue, setSearchValue] = useState(args.value || "");
+    const inputRef = useRef(null);
+
+    const handleChange = (event) => {
+        setSearchValue(event.target.value);
+        args.onChange(event.target.value);
+    };
+
+    const clearValue = () => {
+        setSearchValue("");
+        args.setValue("");
+        inputRef.current.focus();
+    };
+
+    return (
+        <SearchInputBox
```

Comment:
it is not a problem in my opinion, but still I would prefer if you will remove the value controller because it doesn't work

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/49d7dfff-6e8f-46fe-bbb5-5020626a1548

Reply:
Are you talking about the ` value={searchValue}` or something else?

Reply:
yes